step-api-net/StepApiKeyword/Executor.cs

```diff
@@ -146,7 +147,9 @@ public Output CallKeyword(Input input, TokenSession tokenReservationSession, Tok
 
                 try
                 {
+                    script?.BeforeKeyword(method);
                     method.Invoke(c, new object[] { });
+                    script?.AfterKeyword(method);
```

Comment:
@jeromebrongniart The comment of the hook states that the method is called after each keyword invocation. In case of an exception this wouldn't be called. What is the expected behaviour?

Reply:
I did it like that so that only the onError() is called when an error occurs and AfterKeyword only when no error appends.

Reply:
Added a comment on https://github.com/exense/step-api/pull/18/commits/c878e4ee0690404a68dc2410a2e4926e2c263620